sklearn/ensemble/_voting.py

```diff
@@ -91,6 +91,10 @@ class VotingClassifier(ClassifierMixin, _BaseVoting):
         ``self.estimators_``. An estimator can be set to ``None`` or ``'drop'``
```

Comment:
In case we forget in 0.24. Also not  worth advertising a deprecated API anyway. (same below)

```suggestion
         ``self.estimators_``. An estimator can be set to ``'drop'``
```